docs/src/modules/components/overview/MainDemo.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import { useTheme, ThemeProvider, createTheme, Theme, Components } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import Stack from '@mui/material/Stack';
+import Paper from '@mui/material/Paper';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import FlightPicker from './mainDemo/FlightPicker';
+import ThemeToggleGroup from './mainDemo/ThemeToggleGroup';
+import Clock from './mainDemo/Clock';
+import Birthday from './mainDemo/Birthday';
+import DigitalClock from './mainDemo/DigitalClock';
+import DateRangeWithShortcuts from './mainDemo/DateRangeWithShortcuts';
+import PickerButton from './mainDemo/PickerButton';
+import '@mui/x-date-pickers-pro/themeAugmentation';
+
+const components: Components<Theme> = {
```

Comment:
I'll replace this with a new custom theme in a follow-up PR. For now I'll use the `brandingTheme` with some tweaks for demonstration purposes